src/object/mod.rs

```diff
@@ -90,12 +90,14 @@ mod traits {
 
     /// The trait of all object types.
     pub trait Object: Value {
-        fn get<'a, C: Context<'a>, K: PropertyKey>(
+        fn get<'a, C: Context<'a>, K: PropertyKey, V: Value>(
```

Comment:
In other places (e.g., `downcast`), we've put the return type generic in the first position to make turbo fish a little easier.

```rust
// Currently:
let emit = emitter.get::<_, _, JsFunction>(&mut cx, "emit")?;

// Proposed
let emit = emitter.get::<JsFunction, _, _>(&mut cx, "emit")?;
```